deploy/kubernetes/sequencer-service.yaml

```diff
@@ -13,6 +13,9 @@ spec:
   - name: "8080"
     port: 8080
     targetPort: 8080
+  - name: "8081"
```

Comment:
Do you need to export the metrics port on the service too?

Reply:
I think I did?  `sequencer-deployment.yaml` has a new `- containerPort: 8081` line.